docs/BR.md

```diff
@@ -1493,124 +1498,124 @@ Certificates MUST be of type X.509 v3.
 This section specifies the additional requirements for Certificate content and extensions for Certificates generated after the Effective Date.
 
 #### 7.1.2.1 Root CA Certificate
-a. basicConstraints
-This extension MUST appear as a critical extension. The cA field MUST be set true. The pathLenConstraint field SHOULD NOT be present.
+a. `basicConstraints`
```

Comment:
Do we want to revisit how this section is titled in the Markdown ballot? Or are you intentionally deferring the renumbering implications?

That is:
```suggestion
##### 7.1.2.1.1 `basicConstraints`
```